qtof-preprocessing.qmd

```diff
@@ -33,8 +35,11 @@ Next we read the LC-MS/MS data from the mzXML data files.
 sample_data <- read_xlsx("data/qtof_data_files.xlsx") |>
     as.data.frame()
 
+#' We drop the blank data file
+sample_data <- sample_data[sample_data$sample_name != "blank", ]
+
 #' Define the file path where the mzXML files are located
-fp <- "data/mzXML/"
+fp <- "data/mzML/"
```

Comment:
I would suggest you add all files also into the folder of your local clone of the repository, i.e. there to "data/mzML". you have then everything in one folder.